.github/workflows/coverage.yml

```diff
@@ -86,7 +86,7 @@ jobs:
         run: cd "$TMP_DIR/child_build" && mkdir -p Testing/random && touch Coverage.xml
 
       - name: coverage status
-        run: ./scripts/coverage_diff.py "$TMP_DIR/child_build" "$TMP_DIR/parent_build" --pr_id "${{ github.event.number }}"
+        run: ./scripts/coverage_diff.py "$TMP_DIR/child_build" "$TMP_DIR/parent_build" --pr_id "${{ github.event.number }}" --github_token "${{ secrets.GITHUB_TOKEN }}""
```

Comment:
For better security, you can read the token from an environment variable in your script.
```yaml
run: ./scripts/coverage_diff.py "$TMP_DIR/child_build" "$TMP_DIR/parent_build" --pr_id '${{ github.event.number }}'
env:
  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
This way the token does not show up in the process list. This may not matter in practice if the runner is the only thing running on the system, but it is good practice regardless.